modules/external-api/src/main/resources/OSGI-INF/external-events-endpoint.xml

```diff
@@ -47,4 +47,14 @@
             cardinality="1..1"
             policy="static"
             bind="setUrlSigningService" />
+  <reference name="SchedulerService"
```

Comment:
Incorrect number of spaces.  Ugh, this whole file is all out of whack.  Reformat entirely?

Reply:
The formatting is inconsistent not just within this file but rather across all OSGI service declarations of the module. I've created https://opencast.jira.com/browse/MH-12968 for that.

I'm willing to raise another PR that "fixes" MH-12968 (for this module), but I don't think that it makes sense to do this in this PR as this PR is actually not directly related to MH-12968.

That raises a question: How should a OSGI service declaration be formatted? (as this also affects all other modules)

Reply:
See https://github.com/opencast/opencast/pull/314.  I'm assuming that formatting will get rolled into this PR, at least for this file?